hls4ml/writer/quartus_writer.py

```diff
@@ -509,18 +509,21 @@ def write_activation_tables(self, model):
         h_file.close()
 
         ###################
-        ## sigmoid_table
+        # sigmoid_table
         ###################
+        CENTERED = True
+        MAX_VALUE = 8
+        MIN_VALUE = 0
         for layer in model.get_layers():
-            if(layer.get_attr('activation') == 'sigmoid'):
+            if layer.get_attr('activation') == 'sigmoid' and layer.get_attr('table_size') is not None:
```

Comment:
Why not just `table_size = layer.get_attr('table_size', 1024)`?

Reply:
This line allows me to set a different table_size for each activation function, otherwise I would have the same value for both activation functions.